vmdb/app/models/host_openstack_infra.rb

```diff
@@ -1,4 +1,6 @@
 class HostOpenstackInfra < Host
+  has_many                  :openstack_system_services, :dependent => :destroy
```

Comment:
Yes, thanks. Have no idea why I used openstack_system_services.
Consider it done.